app/src/common/shared/com/igalia/wolvic/VersionCheckActivity.java

```diff
@@ -24,11 +17,16 @@ protected void onCreate(Bundle savedInstanceState) {
         if (platformSystemCheck.isOSVersionCompatible()) {
             Intent receivedIntent = getIntent();
             Bundle extras = receivedIntent.getExtras();
+            Uri data = receivedIntent.getData();
 
             // Start VRBrowserActivity if OS version is compatible
             Intent intent = new Intent(this, VRBrowserActivity.class);
-            if (extras != null)
+            if (extras != null) {
                 intent.putExtras(extras);
+            }
+            if (data != null) {
+                intent.setData(data);
+            }
```

Comment:
This looks like a regression of the VersionCheckActivity addition. Should we land it in a different PR?

Reply:
Done in https://github.com/Igalia/wolvic/pull/1575